dlup/experimental_backends/openslide_backend.py

```diff
@@ -40,24 +42,41 @@ def __init__(self, filename: PathLike):
         try:
             mpp_x = float(self.properties[openslide.PROPERTY_NAME_MPP_X])
             mpp_y = float(self.properties[openslide.PROPERTY_NAME_MPP_Y])
+            self.spacing = (mpp_x, mpp_y)
 
         except KeyError:
-            raise UnsupportedSlideError(f"slide property mpp is not available.", str(filename))
+            pass
 
+    @property
+    def spacing(self) -> tuple[float, float] | None:
+        if not self._spacings:
+            return None
+        return self._spacings[0]
+
+    @spacing.setter
+    def spacing(self, value: tuple[float, float]) -> None:
+        if not isinstance(value, tuple) and len(value) != 2:
+            raise ValueError(f"`.spacing` has to be of the form (mpp_x, mpp_y).")
+
+        mpp_x, mpp_y = value
+        check_if_mpp_is_valid(mpp_x, mpp_y)
         mpp = np.array([mpp_y, mpp_x])
-        self._spacings = [tuple(mpp * downsample) for downsample in self.level_downsamples]
+        self._spacings = [cast(tuple[float, float], tuple(mpp * downsample)) for downsample in self.level_downsamples]
```

Comment:
`self._spacings`: I'd prefer to always add the unit of measurement in the name of any quantitative variable; e.g. `self._spacings_mpp` in this case. Can save a lot of headaches and difficult-to-spot bugs.

Reply:
Agreed, but in this case I don't see too much discrepancy. What other spacing would we have?